### PR TITLE
feat: scrim saving

### PR DESCRIPTION
**Pull request by 0c370t**

### CODE REVIEW

**Comment by 0c370t** [comment]
Task linked: [CU-2f3d5y7 Scrim Finalization Process](https://app.clickup.com/t/2f3d5y7)

**Review by gankoji — APPROVED** [review]

File: core/src/replay-parse/finalization/finalization.service.ts [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        private readonly ballchasingConverter: BallchasingConverterService,
+        @InjectConnection() private readonly dbConn: Connection,
+        @InjectRepository(ScrimMeta) private readonly scrimMetaRepo: Repository<ScrimMeta>,
+        @InjectRepository(MatchParent) private readonly matchParentRepo: Repository<MatchParent>,
+        @InjectRepository(Match) private readonly matchRepo: Repository<Match>,
+        @InjectRepository(Round) private readonly roundRepo: Repository<Round>,
+        @InjectRepository(PlayerStatLine) private readonly playerStatRepo: Repository<PlayerStatLine>,
+        @InjectRepository(TeamStatLine) private readonly teamStatRepo: Repository<TeamStatLine>,
+    ) {
+    }
```
Comment (bwpikaard) [inline core/src/replay-parse/finalization/finalization.service.ts:35]: Please remove new line and double or more new lines in the rest of the file.

File: core/src/replay-parse/replay-parse.module.ts [hunk 2]
```diff
@@ -5,13 +5,16 @@ import {
...
+import {MledbInterfaceModule} from "../mledb";
 import {ScrimModule} from "../scrim";
+import {FinalizationService} from "./finalization/finalization.service";
 import {ReplayParsePubSub} from "./replay-parse.constants";
 import {ReplayParseResolver} from "./replay-parse.resolver";
 import {ReplayParseService} from "./replay-parse.service";
 import {ReplayParseSubscriber} from "./replay-parse.subscriber";
 import {ReplayRatificationResolver} from "./replay-ratification/replay-ratification.resolver";
 import {ReplaySubmissionService} from "./replay-submission";
+import { BallchasingConverterService } from './finalization/ballchasing-converter/ballchasing-converter.service';
```
Comment (bwpikaard) [inline core/src/replay-parse/replay-parse.module.ts:17]: Lint please. Also, barrel file?

File: core/src/replay-parse/replay-submission/replay-submission.service.ts [hunk 3]
```diff
@@ -238,6 +196,7 @@ export class ReplaySubmissionService {
 
             this.logger.debug(`Submission ${submissionId} completed and removed`);
 
+
```
Comment (bwpikaard) [inline core/src/replay-parse/replay-submission/replay-submission.service.ts:199]: Useless line

**Review by bwpikaard — CHANGES_REQUESTED** [review]

**Pull request merged**